assets/js/romo/picker.js

```diff
@@ -50,7 +51,18 @@ RomoPicker.prototype.doRefreshUI = function() {
 }
 
 RomoPicker.prototype.doSetValue = function(value) {
-  this.romoOptionListDropdown.doSetNewValue(value);
+  $.ajax({
+    type:    'GET',
+    url:     this.elem.data('romo-picker-url'),
+    data:    { 'value': value },
+    success: $.proxy(function(data, status, xhr) {
+      this.doSetSelectedValueAndText(data[0].value, data[0].displayText);
```

Comment:
@jcredding yeah good call.  I was thinking I'd just assume you'd always get something for a preset value but that probably isn't good at this component level.